commands/storage/delete_bucket_node.go

```diff
@@ -73,7 +75,8 @@ func (c DeleteBucketNodeRecursivelyCommand) Run() ICommandError {
 	}).Try()
 
 	if transactionError != nil {
-		return NewError(http.StatusInternalServerError, transactionError.Error())
+		err := errors.New("error while executing transaction")
+		return NewError(http.StatusInternalServerError, err)
```

Comment:
This should stay as it was: A transaction runs multiple commands, and return the error thrown by the first command that failed. This allows better precision.

However, you can change the `http.StatusInternalServerError` to `transactionError.Code()`, as the precise error code is also returned by commands.